packages/core/src/BlockNoteTheme.ts

```diff
@@ -0,0 +1,131 @@
+      styles: (theme) => ({
+        root: {
+          ...theme.other.defaultMenuStyles(theme),
+          gap: "4px",
+          minWidth: "145px",
+          // Row
+          ".mantine-Group-root": {
+            flexWrap: "nowrap",
+            gap: "8px",
+            paddingInline: "6px",
+            // Row icon
+            ".mantine-Container-root": {
+              color: theme.colors.brandFinal,
+              display: "flex",
+              justifyContent: "center",
+              padding: "0",
+              width: "fit-content",
+            },
+            // Row input field
+            ".mantine-TextInput-root": {
+              background: "transparent",
+              width: "300px",
+              ".mantine-TextInput-wrapper": {
+                ".mantine-TextInput-input": {
+                  fontSize: "12px",
+                  border: 0,
+                  padding: 0,
+                },
+              },
+            },
+          },
+        },
+      }),
+    },
+    Toolbar: {
+      styles: (theme) => ({
+        root: {
+          ...theme.other.defaultMenuStyles(theme),
+          flexWrap: "nowrap",
+          gap: "2px",
+          width: "fit-content",
+          // Button (including dropdown target)
+          ".mantine-UnstyledButton-root": {
+            borderRadius: "4px",
+          },
+          // Dropdown
+          ".mantine-Menu-dropdown": {
+            ...theme.other.defaultMenuStyles(theme),
+            // Dropdown item
+            ".mantine-Menu-item": {
+              borderRadius: "4px",
+              color: theme.colors.brandFinal,
+              fontSize: "12px",
+              height: "34px",
+              ".mantine-Menu-itemRightSection": {
+                paddingLeft: "5px",
+              },
+            },
+          },
+        },
+      }),
+    },
+    SuggestionList: {
+      styles: (theme) => ({
+        root: {
+          ...theme.other.defaultMenuStyles(theme),
+          ".mantine-Menu-item": {
+            // Icon
+            ".mantine-Menu-itemIcon": {
+              padding: "8px",
+              border: `1px solid ${theme.colors.brandFinal[2]}`,
+              backgroundColor: theme.colors.brandFinal[0],
+              borderRadius: "4px",
+              color: theme.colors.brandFinal,
+            },
+            // Text
+            ".mantine-Menu-itemLabel": {
+              color: theme.colors.brandFinal,
+              paddingRight: "16px",
+              ".mantine-Stack-root": {
+                gap: "0",
+              },
+            },
+            // Badge (keyboard shortcut)
+            ".mantine-Menu-itemRightSection": {
+              ".mantine-Badge-root": {
+                border: `1px solid ${theme.colors.brandFinal[2]}`,
+              },
+            },
+          },
+        },
+      }),
+    },
+  },
+  fontFamily: "Inter",
+  primaryColor: "brandFinal",
+  primaryShade: 9,
+  other: {
+    defaultMenuStyles: (theme: MantineTheme) => ({
```

Comment:
Can't we just set this in `{ Component: Menu {...} }`?

Reply:
For the primaryColor & primaryShade, they're supposed to get applied by default to stuff like buttons but it's not really working anyway so tbh I'll probably remove them.

The styles created by the defaultMenuStyles function are used in a bunch of different menus, so it's pretty much just there to reduce code duplication.

Reply:
But can't we target those menus in the normal Mantine way? `{ components: Menu {...} }` would target all Menus right? Why not add the styles to that?

Reply:
Yep, you're right, I thought I might be using different menu styles somewhere else but it seems like all menus use the same styles.

Reply:
Ah, I forgot that the Toolbar component a Group element instead of a Menu.dropdown like the suggestions, headings, and drag handle menus. So I can put the styles in Menu.dropdown for those but I would have to duplicate them in Toolbar.root. Should I do this or keep it as is?

Reply:
yes let's do it I think